politeiawww/api/v1/api.md

```diff
@@ -1473,6 +1466,185 @@ Reply:
 }
 ```
 
+### `Proposal start vote compact`
```

Comment:
Start vote compact should not exist. By definition, this command requires admin access and needs to save the answer. We talked on slack about the need for a short status command so we should rename this compact to VoteStatus using the votestatus route and it should return one of the following answers:
```
const (
 ErrorStatusInvalid                     ErrorVoteStatusT = 0
 ErrorStatusNotStarted                  ErrorVoteStatusT = 1
 ErrorStatusStarted                     ErrorVoteStatusT = 2
 ErrorStatusFinished                    ErrorVoteStatusT = 3
 ErrorStatusDoesntExist                 ErrorVoteStatusT = 4
)
```

Also add a corresponding map to translate the numerical errors to human friendly string errors.